lib/internal/errors.js

```diff
@@ -234,3 +248,22 @@ function oneOf(expected, thing) {
     return `of ${thing} ${String(expected)}`;
   }
 }
+
+function valueOutOfRange(name, compareWord, target, actual) {
```

Comment:
I'm not convinced this is not overkill... For example https://github.com/nodejs/node/blob/master/lib/readline.js#L109
Can you try and find any more places where it's useful?
Otherwise a simpler error message should be enough.